compiler/codegen/Relocation.cpp

```diff
@@ -570,7 +570,7 @@ TR::IteratedExternalRelocation::IteratedExternalRelocation(uint8_t *target, uint
 void TR::IteratedExternalRelocation::initializeRelocation(TR::CodeGenerator *codeGen)
    {
    AOTcgDiag0(TR::comp(), "TR::IteratedExternalRelocation::initializeRelocation\n");
-   _relocationDataCursor = codeGen->getAheadOfTimeCompile()->initializeAOTRelocationHeader(this);
+   _relocationDataCursor = codeGen->getAheadOfTimeCompile()->self()->initializeAOTRelocationHeader(this);
```

Comment:
`self()` isn't be needed here.  It is only need to access member functions in a class hierarchy from within that class hierarchy.

Reply:
Removed that, thank you!